Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1715,7 +1716,7 @@ namespace Multiplayer
                 mutableAddress[portSeparator] = '\0';
                 const char* addressStr = mutableAddress;
                 const char* portStr = &(mutableAddress[portSeparator + 1]);
-                int32_t portNumber = static_cast<int32_t>(atol(portStr));
+                const int32_t portNumber = static_cast<int32_t>(atol(portStr));
```

Comment:
While you're here, aznumeric_cast should be used in this case.

Reply:
Oh good call.